components/story/Story.tsx

```diff
@@ -0,0 +1,44 @@
+import { StatusBar } from "expo-status-bar";
+import React from "react";
+import { StyleSheet, View } from "react-native";
+import ProfilePictre from "../profile-picture/ProfilePicture";
+import StoryText from "./story-text/StoryText";
+
+type StoryProps = {
+  text: string;
+  imageSize: number;
+};
+
+const REDGRADIENT = ["#FD1D1D", "#F56040", "#F77737", "#FCAF45", "#FFDC80"];
+const LIGHTGREY = ["#d3d3d3", "#d3d3d3"];
+const TRASNPARENT = ["transparent", "transparent"];
+
+const openStories = () => {
+  // do something
+};
```

Comment:
`Story` should not care handle press. It is not its responsibility. 
It should simply receive `onPress` prop and call it.